deepedit/ignite/train.py

```diff
@@ -0,0 +1,460 @@
+import argparse
```

Comment:
@wyli Do we require to add the copyright for python code?

Thanks.

Reply:
Just added the copyright